README.md

```diff
@@ -26,17 +32,20 @@ Here's how to get started with pgai:
      a pre-built Docker developer environment with a self-hosted Postgres instance with pgai and our vectorizer worker 
      installed. This takes less than 10 minutes!
```

Comment:
I personally would prefer to have a separate section (further up) that highlights the different entrypoints:
- Self-hosted
- Trial Cloud sign-up (and maybe mention our sample data flow)

Right now this whole section is a big blue link flood and I wouldn't know where to start:
<img width="500" alt="Screenshot 2024-10-25 at 08 17 16" src="https://github.com/user-attachments/assets/ae9751ac-2a13-491c-ae55-5254c7d660eb">

Compared to this which gives the user a much more clear directions imo:
<img width="500" alt="Screenshot 2024-10-25 at 08 21 12" src="https://github.com/user-attachments/assets/80232943-7b39-42cd-baf7-9e7fd77f6493">

Additionally, we could have a demo video at the top that runs through the selfhosted-quickstart or our cloud flow.

Attention spans are short, we should really show people within seconds what this repository is about and what it can do.